.github/workflows/django-tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8"]
+        python-version: ["3.11"]
```

Comment:
why not keep test for all versions between 3.8 and 3.11? or at least 3.10 and 3.11

Reply:
fixed